73-ballan-wrasse/technical_specification.md

```diff
@@ -0,0 +1,70 @@
+# Access Request Management Improvements (Ballan Wrasse)
+
+**Epic Type:** Implementation Epic
+
+Epic planning and implementation follow the
+[Epic Planning and Marathon SOP](https://docs.ghga-dev.de/main/sops/sop001_epic_planning.html).
+
+## Scope
+
+### Outline
+
+The goal of this epic is to improve the management of access requests by giving
+data stewards control over the validity period and enriching the data access request with existing information (e.g., the corresponding DAC) and additional data entered
+by the data stewards (e.g., reason for rejecting a request, internal notes, ticket ID).
+
+### Included/Required
+
+- Data stewards should be able to modify the requested validity period
+  when the request is approved.
+- Data stewards should be able to add a note to the requester
+  (e.g., reason for denial of an access request or modified validity period).
+- Data stewards should be able to add a ticket ID and internal notes.
```

Comment:
Could we make this 'ticket ID' an autoincremental number? I don't see a point in having custom IDs.

Reply:
Ticket ID refers to the ID that is created by the help desk where the access request is further processed. From this ID we can then derive the link to the ticket in the help desk.

Reply:
I have made explained this a bit more now.

Reply:
I understand, but with custom IDs, we need to check existing, process inputs, etc. A simple auto-generated auto-incremental ID (e.g. YouTrack tasks) might be better, in my opinion.

Reply:
The ticket ID is generated in the help desk, so we have no control over it. We do not need to check whether it's existing etc. It just helps the data steward to find the associated help desk ticket.